assets/release/Dockerfiles/Dockerfile.ubuntu-22.04

```diff
@@ -54,7 +58,9 @@ RUN apt-get install -y \
     rustup target add wasm32-unknown-unknown && \
     curl -sLO https://golang.google.cn/dl/go${GOLANG_VERSION}.linux-$(cat /TG_ARCH).tar.gz && \
     sudo rm -rf /usr/local/go && tar -C /usr/local -xzf go${GOLANG_VERSION}.linux-$(cat /TG_ARCH).tar.gz && \
-    curl -sLO https://github.com/tinygo-org/tinygo/releases/download/v$TINYGO_VERSION/tinygo_${TINYGO_VERSION}_$(cat /TG_ARCH).deb && \
+    curl -sLO https://github.com/tinygo-org/tinygo/releases/download/v${TINYGO_VERSION}/tinygo_${TINYGO_VERSION}_$(cat /TG_ARCH).deb && \
     dpkg -i tinygo_${TINYGO_VERSION}_$(cat /TG_ARCH).deb && \
+    curl -sLO https://nodejs.org/dist/v${NODEJS_VERSION}/node-v${NODEJS_VERSION}-linux-$(cat /NODEJS_ARCH).tar.xz && \
+    tar -C /usr/local --strip-components=1 -xf node-v${NODEJS_VERSION}-linux-$(cat /NODEJS_ARCH).tar.xz && \
```

Comment:
Could you uncomment the section in `release.yml` so that this PR can trigger a release job? That way we can test if the PR doesn't break the release pipeline.